web-app/src/pages/User/styled.js

```diff
@@ -22,7 +22,7 @@ export const Container = styled.div`
     margin-bottom: 5px;
   }
   h2 {
-    margin: 0;
+    margin: 0 0 1rem 0;
```

Comment:
Some are .rem and some are .px is there a reason for that?

Reply:
No reason! Will change.